src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -122,6 +122,13 @@ public void remove(ReadOnlyPerson toRemove) throws PersonNotFoundException {
         }
     }
 
+    /**
+     * Sort the list by name in alphabetical order.
+     */
+    public void sortName() {
+        internalList.sort((Person p1, Person p2)->p1.getName().toString().compareToIgnoreCase(p2.getName().toString()));
+    }
+
```

Comment:
Interesting that you chose to implement `sort` such that it modifies the internal state of the address book. Do you think there is another simpler way to sort?